config/i18n-tasks.yml

```diff
@@ -95,6 +95,8 @@ search:
 ## Consider these keys used:
 ignore_unused:
   - 'mail_form.attributes.contact_form.{email,message,name,nickname}'
+  - 'mail_form.attributes.business_contact_form.{company_name,email,first_name,industry,last_name,message,phone_number}'
+  - 'business_contact_forms.areas_of_interest.{infrastructure_development,mobile_application_development,other,product_design,web_application_development}'
```

Comment:
I'm pretty sure we don't need this, we just need to make some changes to the form.